web/app/js/components/Sidebar.jsx

```diff
@@ -1,11 +1,60 @@
+import _ from 'lodash';
+import { ApiHelpers } from './util/ApiHelpers.js';
 import { Link } from 'react-router-dom';
 import logo from './../../img/reversed_logo.png';
-import { Menu } from 'antd';
 import React from 'react';
 import Version from './Version.jsx';
+import { AutoComplete, Menu } from 'antd';
 import './../../css/sidebar.css';
 
+const searchBarWidth = 240;
+
 export default class Sidebar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.api = ApiHelpers(this.props.pathPrefix);
+    this.filterDeployments = this.filterDeployments.bind(this);
+    this.onAutocompleteSelect = this.onAutocompleteSelect.bind(this);
+    this.loadFromServer();
+
+    this.state = {
+      autocompleteValue: '',
+      deployments: []
+    };
+  }
+
+  loadFromServer() {
+    this.api.fetchPods().then(r => {
+      let deploys =  _(r.pods)
+        .groupBy('deployment')
+        .keys()
+        .sort().value();
```

Comment:
oh yeah, that is a good idea! thanks!

Reply:
we remove the conduit deployments from the general list in Deployments too. it might be nice to make the conduit deployment pages accessible from /somewhere/  but removing it here is probably a good idea because the `conduit/` names will tend to show up near the top alphabetically and will therefore take up space...